### DISCUSSION TITLE
关于Alist for ios

### DISCUSSION THREAD

**Discussion by vctao** [AlistGo/alist#3345] [General]
是否可以做一个Alist for ios的IPA安装包，这样我就可以在闲置的iPhone上部署它。

**Comment by anwen-anyi** [comment]
在TG已经回答的很清楚啦~

**Reply by ppggff** [reply]
能了解一下么？多谢了

**Comment by vctao** [comment]
已收到您的邮件!

**Comment by vctao** [comment]
已收到您的邮件!

**Comment by xiangdeyue** [comment]
我最近也在找，官网有一个alist-darwin-arm64.tar.gz的包，但是我解压出来签名之后提示下面这个错误：
iPhone:~/alist root# ./alist
dyld: Library not loaded: /System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation
  Referenced from: /private/var/root/alist/./alist
  Reason: image not found
zsh: abort      ./alist

**Reply by pongfcnkl** [reply]
这个是给mac用的
iPhone需要另外编译

**Reply by xiangdeyue** [reply]
我找了好久，网上好像没有人编译过iPhone的，我自己尝试编译，但是能力有限没成功。

**Reply by pongfcnkl** [reply]
你有苹果电脑的话可以参考这个
https://web.archive.org/web/20190821142332/http://www.iikira.com/2017/08/09/golang-compile-jc-1/

**Reply by xiangdeyue** [reply]
好的，谢谢

**Comment by pongfcnkl** [comment]
我尝试编译了ios的二进制
但是我没有ios设备没法测试 
越狱了的测试一下
https://mypikpak.com/s/VNwt7MG-u6hMKPDMJ63IB1Kso1
非越狱设备不用折腾了

**Comment by sunzongzheng** [comment]
可以运行了：https://github.com/sunzongzheng/alist-expo

**Reply by pongfcnkl** [reply]
ios没后台，可以考虑内置播放器
